kyo-scheduler/jvm-native/src/test/scala/kyo/scheduler/WorkerTest.scala

```diff
@@ -18,6 +18,8 @@ import org.scalatest.freespec.AnyFreeSpec
 
 class WorkerTest extends AnyFreeSpec with NonImplicitAssertions {
 
+    val executor = Executors.newCachedThreadPool(Threads("test-worker"))
```

Comment:
This was the hardest case. SN seems to have an issue with shutting down daemon thread pools with active threads (workers in this case)